components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -94,6 +94,7 @@
     ScreenPlateLinkI, AnnotationAnnotationLinkI, TagAnnotationI
 from omero import ApiUsageException, ServerError, CmdError
 from omero.rtypes import rlong, rlist
+from omeroweb.api.views import LoginView
```

Comment:
The base class should be left in the webgateway. Having interdependence between the `api` and `webclient` application seems like a bad idea. Especially if we decide to move the `api` application into its own repository.